src/context.c

```diff
@@ -27,6 +27,20 @@ static gchar *regex_match(const gchar *pattern, const gchar *string)
 	return NULL;
 }
 
+static gchar* get_machine_id(void)
+{
+	gchar *contents = NULL;
+	g_autoptr(GError) ierror = NULL;
+
+	if (!g_file_get_contents("/etc/machine-id", &contents, NULL, &ierror)) {
+		g_warning("Failed to get machine-id: %s", ierror->message);
```

Comment:
This probably shouldn't be a `g_warning` (perhaps `g_info`), as not all targets use systemd. It also breaks the tests, as warning are fatal there.

Reply:
Ack, fixed.